README.md

```diff
@@ -17,5 +17,31 @@ Troubleshooting:
 ## Testing
 1. Run all tests in `./tests/` with `cargo test`
 
+## Running
+
+For the client to interface with TF2 properly, you must have the following options somewhere in your TF2 launch options:
+
+`-condebug -conclearlog -usercon -g15`
+
+And you should add the following lines to your `autoexec.cfg` file or whatever config file you have setup to run on launch.
+```
+ip 0.0.0.0
+rcon_password mac_rcon
+net_start 
+```
+> Note: the rcon_password is subject to change. If you use loopback (127.0.0.1) for the rcon IP, you are prevented from joining community servers.
+
+You will also need to provide a SteamAPI key to the client. The client looks for configs in a config folder specified by OS.
+- Windows: `C:\Users\<Your Name>\AppData\Roaming\Mega Anti Cheat\config`
+- Mac OS: TODO - work out what this is 
+- Linux: `~/.config/megaanticheatclient/config`
```

Comment:
It's not in a folder called config, the config file is just directly in megaanticheatclient, i.e. the config file is at `~/.config/megaanticheatclient/config.yaml`
The same may be true on Windows but I'd need to check.

Reply:
ah miscommunication

because in windows it has a config folder